.github/workflows/run-weekly-newsletter.yaml

```
@@ -0,0 +1,85 @@
+  #   - cron: "*/50 * * * *"
+
+jobs:
+  velog-posts-daily-stats-batch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)